README.md

```diff
@@ -1,8 +1,49 @@
-Site: https://dhis2.github.io/climate-tools
+# Climate Tools
 
-Based on Jupyter Book v2: https://next.jupyterbook.org
+**Climate Tools** is the umbrella project for working with DHIS2 and environmental / climate-related data.  
+It includes online documentation, guides, tutorials, and the `dhis2eo` Python package and CLI tool.
 
-## Run locally
+Documentation site: https://dhis2.github.io/climate-tools
 
-pip install --pre "jupyter-book==2.\*"
-jupyter book start
+---
+
+## Installation
+
+The Python library and command-line utilities are distributed under the package name **`dhis2eo`**. 
+
+To install the climate tools Python library, and its associated commandline utilities, you need to install the `dhis2eo` Python package. 
+
+You can install it directly from source (development mode):
+
+        $ git clone https://github.com/dhis2/climate-tools.git
+        $ cd climate-tools
+        $ pip install -e .
+
+Once installed, you can use both:
+
+- the Python library:
+
+    ```python
+    import dhis2
+    ```
+
+- the CLI tool:
+
+        $ dhis2eo --help
+
+## Documentation
+
+The climate tools documentation also includes a number of guides and examples for common climate operations and workflows, available at https://dhis2.github.io/climate-tools. This online documentation is built using the MyST web publishing ecosystem (https://mystmd.org/guide/quickstart) and Jupyter v2 (https://next.jupyterbook.org). The site is automatically updated by making changes to the `docs` folder and pushing to the `main` branch. 
+
+To preview the documentation locally you need to have `Node.js` (https://nodejs.org) installed. Additionally you need to install the following Python packages: 
+
+        $ pip install mystmd
```

Comment:
Is this required, or is it also part of the jupyter-book package?

Reply:
Probably already included